### PR TITLE
new parser sssd_network_config

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: rasrivas <rasrivas@redhat.com>

### CODE REVIEW

**Review by bfahr — APPROVED** [review]
Approved pending spec approval.

**Comment by xiangce** [comment]
@rasrivas-redhat - as discussed, would you please close this PR ?

**Comment by rasrivas-redhat** [comment]
closing this PR as the spec collected by this parser can be also collected using systemctl_show parser.

**Pull request closed without merging**
